openedx2zim/html_processor.py

```diff
@@ -149,6 +149,18 @@ def download_images_from_html(self, html_body, output_path, path_from_html):
                         img.attrib["style"] = " max-width:100%"
         return bool(imgs)
 
+    def calculate_root_from_asset(self, path_from_html, root_from_html):
```

Comment:
I think `get_root_from_asset` is more appropriate here

Reply:
Yup. Sounds better.